Hybrid/src/main/java/com/microsoft/azure/management/storage/samples/ManageStorageAccount.java

```diff
@@ -0,0 +1,246 @@
+import com.microsoft.azure.management.storage.v2016_01_01.EncryptionServices;
+import com.microsoft.azure.management.storage.v2016_01_01.Kind;
+import com.microsoft.azure.management.storage.v2016_01_01.Sku;
+import com.microsoft.azure.management.storage.v2016_01_01.SkuName;
+import com.microsoft.azure.management.storage.v2016_01_01.StorageAccount;
+import com.microsoft.azure.management.storage.v2016_01_01.StorageAccountKey;
+import com.microsoft.azure.management.storage.v2016_01_01.StorageAccounts;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.client.ClientProtocolException;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.util.EntityUtils;
+import org.json.JSONObject;
+
+/**
+ * Azure Storage sample for managing storage accounts - - Create a storage
+ * account - Get | regenerate storage account access keys - Create another
+ * storage account - Create another storage account - List storage
+ * accounts - Delete a storage account.
+ */
+
+public final class ManageStorageAccount {
+    /**
+     * Main function which runs the actual sample.
+     * 
+     * @param azure instance of the azure client
+     * @return true if sample runs successfully
+     */
+    public static boolean runSample(Azure azure, String location) {
+        final String storageAccountName = Utils.createRandomName("sa");
+        final String storageAccountName2 = Utils.createRandomName("sa2");
+        final String rgName = Utils.createRandomName("rgSTMS");
+        try {
+            // ===========================================================
+            // Create a resource group
+            ResourceGroup resourceGroup = azure.resourceGroups().define(rgName).withExistingSubscription()
+                    .withLocation(location).create();
+            System.out.println("Created a resource group with name: " + resourceGroup.name());
+
+            // ============================================================
+            // Create a storage account
+
+            System.out.println("Creating a Storage Account");
+            StorageAccount storageAccount = azure.storageAccounts().define(storageAccountName).withRegion(location)
+                    .withExistingResourceGroup(rgName).withKind(Kind.STORAGE)
+                    .withSku(new Sku().withName(SkuName.STANDARD_LRS)).create();
+
+            System.out.println("Created a Storage Account:");
+            Utils.print(storageAccount);
+
+            // ============================================================
+            // Get | regenerate storage account access keys
+
+            System.out.println("Getting storage account access keys");
+            List<StorageAccountKey> storageAccountKeys = azure.storageAccounts().inner()
+                    .listKeys(rgName, storageAccountName).keys();
+
```

Comment:
will be good if we use standard methods (though they are async) instead of inner.

```java
List<StorageAccountKey> storageAccountKeys = azure.storageAccounts().listKeysAsync(rgName, storageAccountName)
               .toList().toBlocking().last().get(0).keys();
```